src/site/content/en/payments/registering-a-web-based-payment-app/index.md

```diff
@@ -0,0 +1,321 @@
+object](https://w3c.github.io/payment-handler/#dom-paymentinstrument) that
+contains details about the instrument.
+
+{% Label %}[payment handler] payment app's landing pages{% endLabel %}
+
+```js
+...
+  // Feature detection
+  if (!registration.paymentManager) return;
+
+
+  await registration.paymentManager.instruments.set(
+    // Payment instrument key
+    "bobpay_card_1",
+    // Payment instrument details
+    {
+      // This parameter will be ignored in Chrome
+      name: 'Payment Handler Example',
+      // This parameter will be used to match against
+      // the PaymentRequest.
+      method: 'https://bobpay.xyz/pay'
+    }
+  );
+```
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th colspan="2">Arguments</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td colspan="2">
+          <a href="https://w3c.github.io/payment-handler/#paymentinstruments-interface"><code>instrumentKey</code></a> (required)
+        </td>
+        <td>
+          An arbitrary string used only to identify the instrument when you want to update it. It's not visible to customers and it's recommended you use an identifier from your payment app backend.
+        </td>
+      </tr>
+      <tr>
+        <td rowspan="5">
+          <a href="https://w3c.github.io/payment-handler/#paymentinstrument-dictionary"><code>PaymentInstrument</code></a>
+        </td>
+        <td></td>
+        <td>
+          Details of the payment instrument.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <a href="https://developer.mozilla.org/docs/Web/Manifest/name"><code>name</code></a> (required)
+        </td>
+        <td>
+          Sets a string as the instrument name. Required but ignored in Chrome.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <a href="https://developer.mozilla.org/docs/Web/Manifest/icons"><code>icons</code></a>
+        </td>
+        <td>
+          Sets an array of <a href="https://w3c.github.io/payment-handler/#imageobject-dictionary"><code>ImageObject</code></a>s that the Payment Request sheet will display. Ignored in Chrome.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <code>method</code>
+        </td>
+        <td>
+          A supported payment method identifier.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <code>capabilities</code>
+        </td>
+        <td>
+          Sets the <a href="https://w3c.github.io/payment-handler/#capabilities-example">payment method specific parameters as an object</a>. As of July 2020, basic-card is the only payment method that accepts capabilities.
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+{% Aside %}
+Chrome ignores the `name` and `icons` properties. It respects the [web app
+manifest](/setting-up-a-payment-method/#step-3:-serve-a-web-app-manifest)'s
+respective properties instead, but other browsers may behave differently.
+{% endAside %}
```

Comment:
We don't know how other browsers behave? As a web developer considering Web Payments, I think the story will be much more compelling if this article at least tells me the current state of things in other browsers

Reply:
Again, this API is not implemented in any other browsers yet. Though Firefox just started doing so.